### PR TITLE
chore: Define options for for doubleClick, buttonUp\Down routes

### PR DESCRIPTION
**Pull request by snoopy-peace**
I have added support for POST commands for DoubleClick, ButtonUp and ButtonDown.

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Please update the unit tests

File: lib/protocol/routes.js [hunk 1]
```diff
@@ -252,13 +252,13 @@ const METHOD_MAP = {
     POST: {command: 'clickCurrent', payloadParams: {optional: ['button']}}
   },
   '/session/:sessionId/buttondown': {
-    POST: {}
+    POST: {command: 'buttondown', payloadParams: {optional: ['button']}}
```
Comment (mykola-mokhnach) [inline lib/protocol/routes.js:255]: command: 'buttonDown'

File: lib/protocol/routes.js [hunk 2]
```diff
@@ -252,13 +252,13 @@ const METHOD_MAP = {
...
   },
   '/session/:sessionId/buttonup': {
-    POST: {}
+    POST: {command: 'buttonup', payloadParams: {optional: ['button']}}
```
Comment (mykola-mokhnach) [inline lib/protocol/routes.js:258]: command: 'buttonUp'

**Comment by snoopy-peace** [comment]
> Please update the unit tests

Thanks for reviewing the PR. Could you please highlight where the Unit Tests are present.

**Comment by mykola-mokhnach** [comment]
It is just necessary to update the checksum. See https://github.com/appium/appium-base-driver/pull/405/files#diff-2b61a65d06ba82e0f1a8deca5039baef for example. The actual checksum is printed in the corresponding CI log

**Comment by mykola-mokhnach** [comment]
for the other failing test simply update the message at https://travis-ci.org/github/appium/appium-base-driver/jobs/678550784#L3111

(not sure why it was not failing before)

**Comment by snoopy-peace** [comment]
> https://travis-ci.org/github/appium/appium-base-driver/jobs/678550784#L3111

sorry didn't understand where do I update the message on the mentioned link.

**Comment by snoopy-peace** [comment]
> for the other failing test simply update the message at https://travis-ci.org/github/appium/appium-base-driver/jobs/678550784#L3111
> 
> (not sure why it was not failing before)

@mykola-mokhnach  I tried fixing the test error. At line 208\205 in file **test/protocol/protocol-e2e-specs.js** the message mentioned is the expected one even then the CI tests are failing. 
Also, i tried changing the message at different places corresponding to error '405' even then the tests are failing on the CI. 
Would request you to please check once. Thanks.

**Comment by mykola-mokhnach** [comment]
Update the message at line 208 of that test source

**Comment by snoopy-peace** [comment]
Line 208 already has the expected message
https://github.com/snoopy-peace/appium-base-driver/blob/snjain/routeAPI/test/protocol/protocol-e2e-specs.js#L208

**Pull request merged**